doc/widget-api-frontend.md

```diff
@@ -107,6 +160,8 @@ example.
 The current locale of a field the widget is attached to. Yields `"en_US"` in the
 example.
 
+##### `widget.field.type: string`
+Gives the field type of the widget.
```

Comment:
“Holds the type of the field the widget is attached to”. Please also link to the CMA documentation for a list of all field types.